### DISCUSSION TITLE
Having trouble pulling any metadata when using the Plugin

### DISCUSSION THREAD

**Discussion by a-curious-sprig** [CodyBerenson/PGMA-Modernized#297] [Q&A] [answered]
I have recently gotten plex to manage all my downloaded media and thought itd be fun to organize all my adult videos into my library. Currently I have the most recent version of plex installed on a NAS and for the most part, it works great and the way I'd like it to. However, when it comes to using this plugins agents, it wont fetch any meta data, even after I set up the file names correctly.
![Screenshot 2024-04-27 005336](https://github.com/CodyBerenson/PGMA-Modernized/assets/168264417/2480df09-e39d-4e2c-b63d-bbd1d677b998)
![Screenshot 2024-04-27 005349](https://github.com/CodyBerenson/PGMA-Modernized/assets/168264417/3d347e6e-8583-410e-bcdd-1527d88150ee)
![Screenshot 2024-04-27 005409](https://github.com/CodyBerenson/PGMA-Modernized/assets/168264417/513680a1-4f4d-4fcf-88d4-3d1fa15720b8)
Anyone have a solution?

**Comment by JPH71** [comment]
If you can find the film on either AEBN or GEVI. Just refresh metadata
rather than fix match etc...
That should do the trick.

**Reply by a-curious-sprig** [reply]
I tried and unfortunately, it didn't work :( Any other potential fixes?

**Comment by JPH71** [comment]
I have just scraped the film, and it works
here is the visual and the log file from GEVI
[com.plexapp.agents.AEBN.log](https://github.com/CodyBerenson/PGMA-Modernized/files/15137936/com.plexapp.agents.AEBN.log)
[com.plexapp.agents.GEVI.log](https://github.com/CodyBerenson/PGMA-Modernized/files/15137937/com.plexapp.agents.GEVI.log)
![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/43102249/d1dab3e2-bf5b-4eee-86e4-f7b94e5a4a62)
![image](https://github.com/CodyBerenson/PGMA-Modernized/assets/43102249/65258daa-6ade-4fff-9e08-42c52706479e)

versions
AEBN v.2020.05.21.18
GEVI v.2019.12.25.50

Utility update date: 04 Mar 2024

If it still does not work for you - move the film out of the folder. Rescan the library, then put it back into the library and rescan.

**Comment by CodyBerenson** [comment]
@a-curious-sprig Can you please provide your log for Jason to troubleshoot?  FWIW, I had similar experiences when I tried to use my NAS as both my content respository and my Plex Media Server.  Getting the privileges correct was a nightmare.  Not to mention the container for the PMS grew so big it exploded.  Anyway, the logs will help.

**Comment by JPH71** [comment]
Send the log for gevi!

**Comment by a-curious-sprig** [comment]
Heres the GEVI log.

[com.plexapp.agents.GEVI.log](https://github.com/CodyBerenson/PGMA-Modernized/files/15139699/com.plexapp.agents.GEVI.log)

**Comment by JPH71** [comment, marked as answer]
Your error as per the log on line 285

:    Error: < File Name [(Hollywood Sales) – Lord of the Cock Ring (2002)]
not in the expected format: (Studio) - Title [(Year)] [-
cd|disc|disk|dvd|part|pt|scene 1..8]! >

You used a hyphen rather than a minus
(Hollywood Sales) - Lord of the Cock Ring (2002) - mine
(Hollywood Sales) – Lord of the Cock Ring (2002) - yours

Use this utility to rename multiple files
Advanced Renamer - Batch renaming utility for files and folders
<https://www.advancedrenamer.com/>

Cheers
Jason

**Comment by JPH71** [comment]
once resolved please close issue!

**Comment by a-curious-sprig** [comment]
I noticed that and thought to myself that couldnt possibley the problem haha... It worked thanks!

**Comment by JPH71** [comment]
Brilliant 👏
